### DISCUSSION TITLE
Creating Multiple Pages with Nuxi

### DISCUSSION THREAD

**Discussion by will-i-code420** [nuxt/nuxt#18414] [Questions]
I love the addition of nuxi to add pages/components/etc...  however is it possible to create multiple pages using it???  I know i want multiple routes in say an auth route, but when i try npx nuxi add page "auth/login auth/register", it just creates multiple folders that lead to a Auth.vue at the end and when i tried npx nuxi add page "auth/login" "auth/register" it just made a login.vue file so is it possible to do this as knowing all the pages needing to be made and done in one command is much faster than 1 by 1, i just went into my project folder and didn't bother since it was like 10 pages i needed and copy and paste/clicking was much faster than typing them one by one in the terminal but i would like to do that instead as nuxi is awesome

**Comment by Baroshem** [comment]
Hey,

Nuxi at the moment can add only one element at the time. In order to support adding multiple elements with one command it would have to be added to nuxi as the completely new functionality. 

I could implement that but first would like to get the feedback of the Core Team guys.

@danielroe @manniL what do you think about it?
